refarch-frontend/src/App.vue

```diff
@@ -41,6 +41,7 @@
             v-if="appswitcherBaseUrl"
             :base-url="appswitcherBaseUrl"
             :tags="['global']"
+            icon="$appswitcher"
```

Comment:
as an alternative we could also do

```
:icon="mdiApps"
```

and import the icon here directly instead of defining an global alias.

Reply:
I would prefer the explicit import as this is our suggested way of importing icons when writing SFCs. Does this mean vue-appswitcher should be updated to not include an icon by default anymore and we always have to provide an icon in the application which use vue-appswitcher?

Reply:
We could still think about migrating `vue-appswitcher` to `@mdi/js` to be aligned with the `refarch-templates`. I know the library is not solely for projects based on our reference architecture but I think will still be the main use case so we should implement in with "refarch"-first in mind. This way we could set the default to `mdi-apps` using `@mdi/js` inside the library and not require overriding manually. Do you agree?

Reply:
proposal: https://github.com/it-at-m/appswitcher-vue/issues/419

I still have plenty of refarch applications in production which currently use "@mdi/fonts" and I dont want to _force_ users. With the proposal both options (mdi/fonts or mdi/js would be supported).

Reply:
Ok the proposal sounds fine to me. :)